pyadjoint/tape.py

```diff
@@ -226,10 +308,25 @@ def get_tags(self):
         return tags
 
     def evaluate_adj(self, last_block=0, markings=False):
-        for i in self._bar("Evaluating adjoint").iter(
-            range(len(self._blocks) - 1, last_block - 1, -1)
-        ):
-            self._blocks[i].evaluate_adj(markings=markings)
+        """Evaluate the adjoint of the tape.
+
+        Parameters
+        ----------
+        last_block : int, optional
+            The index of the last block to evaluate. Default 0.
+        markings : bool, optional
+            If True, then each :class:`BlockVariable` of the current block
```

Comment:
```suggestion
            If True, then each `BlockVariable` of the current block
```
I think. You should check the autogenerated docs.